README.md

```diff
@@ -67,6 +67,10 @@ jobs:
 
 **Required:** GitHub App private key.
 
+### `revoke`
+
+**Optional:** Whether to revoke the token when the current job is complete. Default: `"true"`.
```

Comment:
I wonder if `revoke` is the best possible argument for this feature. As the default is true. I think it might be more elegant to have an option that is not set by default at all, and needs to be set to overwrite it.

For example: `do_not_revoke_token_after_job`. Long but clear 🤷🏼

Reply:
or `skip_token_revoke`

Reply:
I wondered the same thing! Either of those suggestions works for me.

The 5 actions I surveyed in https://github.com/github/core-ux/discussions/142#discussioncomment-7177238 (only accessible to Hubbers) didn’t reveal a widely-used name, but I’ll review the actions listed in https://github.com/actions/create-github-app-token/issues/42 too.

Reply:
actually I think we should use dashes for our arguments, it seems to be the convention for `actions/*` actions, e.g. see https://github.com/actions/checkout/. So we would do `skip-token-revoke`

Reply:
@gr2m The [existing inputs](https://github.com/actions/create-github-app-token?tab=readme-ov-file#inputs) use underscores—would we want to migrate those to dashes too?

Reply:
I think they both work the same way in the end, they are translated to `INPUTS_MY_VAR` no matter if the input was `my_var` or `my-var`. But we should double check.

Reply:
> …I’ll review the actions listed in https://github.com/actions/create-github-app-token/issues/42…

- [tibdex/github-app-token](https://github.com/tibdex/github-app-token) — Configurable with `revoke`. Default: Doesn’t revoke token.
- [peter-murray/workflow-application-token-action](https://github.com/peter-murray/workflow-application-token-action) — Doesn’t support revoking tokens.
- [getsentry/action-github-app-token](https://github.com/getsentry/action-github-app-token) — Doesn’t support revoking tokens.
- [microsoftgraph/get-app-token](https://github.com/microsoftgraph/get-app-token) — Doesn’t support revoking tokens.
- [wow-actions/use-app-token](https://github.com/wow-actions/use-app-token) — Configurable with `clean`. Default: Revokes token.
- [navikt/github-app-token-generator](https://github.com/navikt/github-app-token-generator) — Doesn’t support revoking tokens.
- [mercari/github-app-token-generator](https://github.com/mercari/github-app-token-generator) — Doesn’t support revoking tokens.
- [jnwng/github-app-installation-token-action](https://github.com/jnwng/github-app-installation-token-action) — Doesn’t support revoking tokens.

Given the lack of precedent, I’ll update this PR to use `skip-token-revoke`. 👍

Reply:
>  …I’ll update this PR to use `skip-token-revoke`.

Done, via https://github.com/actions/create-github-app-token/pull/54/commits/cde1226b0a4339b235072c308e93066fe1146d2b.

> I think they both work the same way in the end, they are translated to `INPUTS_MY_VAR` no matter if the input was `my_var` or `my-var`. But we should double check.

I tested this, and it turns out they are _not_ translated the same way:
- An input named `my_var` is translated to `INPUTS_MY_VAR`.
- An input named `my-var` is translated to `INPUTS_MY-VAR`.

This is documented in [“Metadata syntax for GitHub Actions > Example: Specifying inputs”](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#example-specifying-inputs):
> When you specify an input in a workflow file or use a default input value, GitHub creates an environment variable for the input with the name `INPUT_<VARIABLE_NAME>`. The environment variable created converts input names to uppercase letters and replaces spaces with `_` characters.
> 
> […]
> 
> For example, if a workflow defined the `num-octocats` and `octocat-eye-color` inputs, the action code could read the values of the inputs using the `INPUT_NUM-OCTOCATS` and `INPUT_OCTOCAT-EYE-COLOR` environment variables.

Reply:
> I tested this, and it turns out they are _not_ translated the same way:
> 
> * An input named `my_var` is translated to `INPUTS_MY_VAR`.
> * An input named `my-var` is translated to `INPUTS_MY-VAR`.

Given this, I'm leaning towards using underscores in the variable for consistency in environment variable names and since we already use underscores for the other inputs. If we migrate the other variables, that will result in a breaking change.

Reply:
> I tested this, and it turns out they are _not_ translated the same way:
> 
> * An input named `my_var` is translated to `INPUTS_MY_VAR`.
> * An input named `my-var` is translated to `INPUTS_MY-VAR`.

Thanks for testing this, @smockle! Given your findings, I'm leaning towards using underscores in the variable name for consistency in both environment variable names and input names. If we migrate the other variables to use hyphens, that will result in a breaking change.

Reply:
> * [tibdex/github-app-token](https://github.com/tibdex/github-app-token) — Configurable with `revoke`. Default: Doesn’t revoke token.

Looks like this is a new feature for this action. And digging a bit deeper, it looks like the documentation [incorrectly states](https://github.com/tibdex/github-app-token/pull/95/files) that the value of `revoke` is false by default. In fact, the `revoke` option appears to be true by default. 

I'm in favor of using `revoke` in the same way, but I don't feel strongly about it, so I'll leave it up to @gr2m.

Reply:
okay let's go with `skip_token_revoke`, let's have a separate discussion about whether to migrate arguments to dashes. I'd rather change it now when adaption is still early

Reply:
Sounds good! Just throwing out one more possible alternative name: `do_not_revoke`. 

Take it or leave it, I don't feel strongly about this one either. 😄

Reply:
> …let's have a separate discussion about whether to migrate arguments to dashes…
Opened https://github.com/actions/create-github-app-token/issues/57 for this! 👍

Reply:
> …let's go with `skip_token_revoke`…

Done, via https://github.com/actions/create-github-app-token/pull/54/commits/5a2e1e70261da711ab24a23328f6cf86411dd194.